Sources/ClientRuntime/Networking/Http/CRT/SDKDefaultIO.swift

```diff
@@ -25,13 +19,24 @@ public final class SDKDefaultIO: @unchecked Sendable {
     /// The setter for changing log level of SDKDefaultIO logger.
     /// If any log level other than the default log level of `.none` is desired,
     /// this setter needs to be called as the first thing in the program.
-    public func setLogLevel(level: LogLevel) {
-        Self.setupLogger(level: level)
+    public static func setLogLevel(level: LogLevel) {
+        do {
+            try Logger.initialize(target: .standardOutput, level: level)
+        } catch {
+            failOnLogger()
+        }
     }
 
     private init() {
         CommonRuntimeKit.initialize()
-        Self.setupLogger(level: .none)
+        do {
+            try Logger.initialize(target: .standardOutput, level: .none)
+        } catch CommonRunTimeError.crtError(let error)
+                    where error.code == 6 && error.name == "AWS_ERROR_UNSUPPORTED_OPERATION" {
+            // logger was already initialized, no need to initialize it
```

Comment:
Catch the "logger already initialized" error and ignore it if the logger was already initialized when the log level was set.

Reply:
Does this mean the log level can't be changed after it's been initialized?

Reply:
Discussed offline, this is due to design of CRT logger